tests/raster/mosaic/geotiff/test_geotiff.py

```diff
@@ -0,0 +1,71 @@
+from veranda.raster.mosaic.geotiff import GeoTiffReader, GeoTiffWriter
+from tests.raster.mosaic.mosaic_common import *
```

Comment:
making the `tests` folder a module can cause problems for instance if people don't run the tests from the same working directory. If you want to add common helper functions to your test suite it is better to have them in a common folder which is then added to the path environment variable in conftest.py like so:
```
import sys
from pathlib import Path

import pytest

sys.path.append((Path(__file__).parent / "helpers").as_posix())
```
The helpers folder now can contain mosaic_common.py and you can import function from there like so:
`from mosaic_common import assert_reader_data`

Reply:
Fixed it.